# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libpciaccess/pathmax.patch
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

E.g. for musl.

--- libpciaccess-0.13.4/src/linux_sysfs.c.vanilla	2016-12-24 14:15:50.139685518 +0000
+++ libpciaccess-0.13.4/src/linux_sysfs.c	2016-12-24 14:16:00.635685698 +0000
@@ -48,6 +48,7 @@
 #include <sys/mman.h>
 #include <dirent.h>
 #include <errno.h>
+#include <limits.h>
 
 #if defined(__i386__) || defined(__x86_64__) || defined(__arm__)
 #include <sys/io.h>
